rules/mito_variants.smk

```diff
@@ -0,0 +1,63 @@
+import pandas as pd
+from datetime import datetime
+
+configfile: "config.yaml"
```

Comment:
Is this necessary here? In the future we'll have two configs (one for hpf, the other for CHEO-RI), so I don't want this hard-coded.

Reply:
That's a good point! I was initially running into some issues (can't remember what it was exactly) which is why I added this in, but I can look more into this.